core-bundle/src/Twig/Inheritance/DynamicIncludeTokenParser.php

```diff
@@ -51,12 +52,14 @@ public function parse(Token $token): IncludeNode
      */
     public static function adjustTemplateName(string $name, TemplateHierarchyInterface $hierarchy): string
     {
-        if (null === ($shortNameOrIdentifier = TokenParserHelper::getContaoTemplate($name))) {
+        $parts = ContaoTwigUtil::parseContaoName($name);
+
+        if ('Contao' !== ($parts[0] ?? null)) {
```

Comment:
Why this check? If `$parts` is not `null`, `$parts[0]` will always be `Contao`, won't it?

Reply:
see https://github.com/contao/contao/pull/3201/files#r678459280